.github/workflows/translation.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v4
+        uses: peter-evans/create-pull-request@38e0b6e68b4c852a5500a94740f0e535e0d7ba54 # v4.2.4
```

Comment:
We are using `peter-evans/create-pull-request@6d6857d36972b65feb161a90e484f2984215f83e # v6.0.5` in a different workflow. Can we align those to use the same versions?

Reply:
Nice catch!

I updated to v6 in the translation workflow in a follow-up commit.
Now we should be using consistent versions across all workflows for each action.